code/unclassified/prime_factors

```diff
@@ -0,0 +1,18 @@
+void find_prime_factor(int n){
```

Comment:
Kindly write complete code with header and main function 👍

Reply:
thank you very much :)